contracts/extensions/votingReputation/VotingReputationStorage.sol

```diff
@@ -250,13 +250,14 @@ contract VotingReputationStorage is ColonyExtension, BasicMetaTransaction, Votin
     }
   }
 
-function getActionSummary(bytes memory action, address target) public view returns (ActionSummary memory) {
+  function getActionSummary(bytes memory _action, address _altTarget) public view returns (ActionSummary memory) {
+    address target = getTarget(_altTarget);
```

Comment:
It was a stylistic choice, as I didn't like how verbose most of the calls became when I had to include `getTarget`. Essentially I moved the calls to `getTarget` into the function body, whereas before it had been the responsibility of the calling function.

Reply:
I misread the diff, I understand and agree with putting this call here.

> The changes to the test reflect the fact that an altTarget of 0x0 is interpreted as the colony address.

Right, but the colony address was already being passed in, so they could have also been left as-is, right?

Reply:
Could have, but that would have been a hack as we don't allow `altTarget` to be the colony. So passing in `ZERO_ADDRESS` is technically the correct usage, since it gets converted into the colony address by `getTarget`. That whole functionality is a bit odd frankly (was meant as a gas optimisation), but so it goes.